python/cudf/cudf/_lib/column.pyi

```diff
@@ -32,94 +32,46 @@ class Column:
         null_count: int = None,
         children: Tuple[ColumnBase, ...] = (),
-    ) -> None:
-        ...
-
+    ) -> None: ...
     @property
-    def base_size(self) -> int:
-        ...
-
+    def base_size(self) -> int: ...
     @property
-    def dtype(self) -> DtypeObj:
-        ...
-
+    def dtype(self) -> DtypeObj: ...
     @property
-    def size(self) -> int:
-        ...
-
+    def size(self) -> int: ...
     @property
-    def base_data(self) -> Optional[Buffer]:
-        ...
-
+    def base_data(self) -> Optional[Buffer]: ...
     @property
-    def base_data_ptr(self) -> int:
-        ...
-
+    def base_data_ptr(self) -> int: ...
     @property
-    def data(self) -> Optional[Buffer]:
-        ...
-
+    def data(self) -> Optional[Buffer]: ...
     @property
-    def data_ptr(self) -> int:
-        ...
-
-    def set_base_data(self, value: Buffer) -> None:
-        ...
-
+    def data_ptr(self) -> int: ...
+    def set_base_data(self, value: Buffer) -> None: ...
     @property
-    def nullable(self) -> bool:
-        ...
-
-    def has_nulls(self, include_nan: bool=False) -> bool:
-        ...
-
+    def nullable(self) -> bool: ...
+    def has_nulls(self, include_nan: bool = False) -> bool: ...
     @property
-    def base_mask(self) -> Optional[Buffer]:
-        ...
-
+    def base_mask(self) -> Optional[Buffer]: ...
     @property
-    def base_mask_ptr(self) -> int:
-        ...
-
+    def base_mask_ptr(self) -> int: ...
     @property
-    def mask(self) -> Optional[Buffer]:
-        ...
-
+    def mask(self) -> Optional[Buffer]: ...
     @property
-    def mask_ptr(self) -> int:
-        ...
-
-    def set_base_mask(self, value: Optional[Buffer]) -> None:
-        ...
-
-    def set_mask(self: T, value: Optional[Buffer]) -> T:
-        ...
-
+    def mask_ptr(self) -> int: ...
+    def set_base_mask(self, value: Optional[Buffer]) -> None: ...
+    def set_mask(self: T, value: Optional[Buffer]) -> T: ...
     @property
-    def null_count(self) -> int:
-        ...
-
+    def null_count(self) -> int: ...
     @property
-    def offset(self) -> int:
-        ...
-
+    def offset(self) -> int: ...
     @property
-    def base_children(self) -> Tuple[ColumnBase, ...]:
-        ...
-
+    def base_children(self) -> Tuple[ColumnBase, ...]: ...
     @property
-    def children(self) -> Tuple[ColumnBase, ...]:
-        ...
-
-    def set_base_children(self, value: Tuple[ColumnBase, ...]) -> None:
-        ...
-
-    def _mimic_inplace(self, other_col: ColumnBase, inplace=False) -> Optional[ColumnBase]:
-        ...
-
+    def children(self) -> Tuple[ColumnBase, ...]: ...
+    def set_base_children(self, value: Tuple[ColumnBase, ...]) -> None: ...
+    def _mimic_inplace(
+        self, other_col: ColumnBase, inplace=False
+    ) -> Optional[ColumnBase]: ...
     @staticmethod
-    def from_scalar(
-        val: ScalarLike,
-        size: int
-    ) -> ColumnBase:  # TODO: This should be Scalar, not ScalarLike
```

Comment:
@vyasr FYI this reformat drops comments in `.pyi` files. I noticed this in a previous attempt to update `black` but I wasn't sure how to address it.

Reply:
Thanks for the note. I'll see what we can do about that.

Reply:
It looks like the issue is specifically the inline comments on lines that get compressed. I guess we'll just have to remember to put comments on separate lines from now on in pyi files.

Reply:
With `TODO`s, wonder if we should just file these as issues. That would make them easier to track, cross-reference, triage, resolve, etc.

Reply:
> ...at the moment it's not uncommon for TODOs like this to become moot when the relevant code paths are removed wholesale

Honestly this is my biggest worry with `TODO`s and have seen this happen to other code bases. Comments left that no longer pertain to the code they are near with no one that has a clue what they mean. It is clearer to see what issues intended and when they are resolved.